src/test/java/bio/terra/common/ValidationUtilsTest.java

```diff
@@ -1,31 +1,58 @@
 package bio.terra.common;
 
 import bio.terra.common.category.Unit;
+import org.apache.commons.lang3.RandomStringUtils;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
-import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringRunner;
-
-@RunWith(SpringRunner.class)
-@SpringBootTest
-@AutoConfigureMockMvc
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
 @Category(Unit.class)
 public class ValidationUtilsTest {
 
     @Test
     public void testEmailFormats() throws Exception {
-        assert ValidationUtils.isValidEmail("john@somewhere.com");
-        assert ValidationUtils.isValidEmail("john.foo@somewhere.com");
-        assert ValidationUtils.isValidEmail("john.foo+label@somewhere.com");
-        assert ValidationUtils.isValidEmail("john@192.168.1.10");
-        assert ValidationUtils.isValidEmail("john+label@192.168.1.10");
-        assert ValidationUtils.isValidEmail("john.foo@someserver");
-        assert ValidationUtils.isValidEmail("JOHN.FOO@somewhere.com");
-        assert !ValidationUtils.isValidEmail("@someserver");
-        assert !ValidationUtils.isValidEmail("@someserver.com");
-        assert !ValidationUtils.isValidEmail("john@.");
-        assert !ValidationUtils.isValidEmail(".@somewhere.com");
+        assertThat(ValidationUtils.isValidEmail("john@somewhere.com")).isTrue();
+        assertThat(ValidationUtils.isValidEmail("john.foo@somewhere.com")).isTrue();
+        assertThat(ValidationUtils.isValidEmail("john.foo+label@somewhere.com")).isTrue();
+        assertThat(ValidationUtils.isValidEmail("john@192.168.1.10")).isTrue();
+        assertThat(ValidationUtils.isValidEmail("john+label@192.168.1.10")).isTrue();
+        assertThat(ValidationUtils.isValidEmail("john.foo@someserver")).isTrue();
+        assertThat(ValidationUtils.isValidEmail("JOHN.FOO@somewhere.com")).isTrue();
+        assertThat(ValidationUtils.isValidEmail("@someserver")).isFalse();
+        assertThat(ValidationUtils.isValidEmail("@someserver.com")).isFalse();
+        assertThat(ValidationUtils.isValidEmail("john@.")).isFalse();
+        assertThat(ValidationUtils.isValidEmail(".@somewhere.com")).isFalse();
+    }
+
+    @Test
+    public void testHasDuplicates() {
+        assertThat(ValidationUtils.hasDuplicates(Arrays.asList("a", "a", "b", "c"))).isTrue();
```

Comment:
Now that we're using JDK 11, I think a better way to write this is 
```suggestion
        assertThat(ValidationUtils.hasDuplicates(List.of("a", "a", "b", "c"))).isTrue();
```

Reply:
Ah right!  ok, I'll update all of these